inference/models/rtdetr/rtdetr.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import torch
+import requests
+
+from typing import Any, Tuple
+
+from PIL import Image
+from transformers import RTDetrForObjectDetection, RTDetrImageProcessor
+
+from inference.core.models.base import PreprocessReturnMetadata
+from inference.models.transformers.transformers import TransformerModel
+from inference.core.utils.image_utils import load_image_rgb
+
+DEVICE = "cuda:0" if torch.cuda.is_available() else "cpu"
+
+class RTDETR(TransformerModel):
+    def __init__(self, *args, model_id=f"", **kwargs):
+        super().__init__(*args, model_id=model_id, **kwargs)
+        self.model_id = model_id
+        self.endpoint = model_id
+        self.api_key = API_KEY
+        self.dataset_id, self.version_id = model_id.split("/")
+        self.cache_dir = os.path.join(MODEL_CACHE_DIR, self.endpoint + "/") # "PekingU/rtdetr_r50vd"
+        dtype = torch.bfloat16
```

Comment:
bfloat16 shouldn't be hardcoded here, bfloat16 is only supported on gpus with compute capability >= 8.0